src/utility/html.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * This helper does nothing but interpolate a tagged template literal like
+ * normal. The trick is that it being called html allows Atom’s language-babel
+ * package, and others like it, to use HTML syntax highlighting and indentation
+ * rules inside the literals.
+ *
+ * @providesModule html
+ * @flow
+ */
+
+// - strings is an array of the portions around the interpolations, like what
+//   you would get if you split the string on ${...}
+// - interpolations is an array of the portions inside the interpolations.
+//
+// strings has length >= 1 and interpolations will always be one element shorter
+// than strings
+export default function html(
```

Comment:
This is clever 😹

Reply:
Thanks :) I'm helpless without auto-indent behavior haha.

<img width="906" alt="screen shot 2018-01-18 at 10 52 27" src="https://user-images.githubusercontent.com/4642599/35195961-193f79b4-fe91-11e7-90fe-158561e46c2f.png">